src/main/java/com/aws/greengrass/clientdevices/auth/api/ClientDevicesAuthServiceApi.java

```diff
@@ -11,6 +11,7 @@
 import com.aws.greengrass.clientdevices.auth.exception.AuthenticationException;
 import com.aws.greengrass.clientdevices.auth.exception.AuthorizationException;
 import com.aws.greengrass.clientdevices.auth.exception.CertificateGenerationException;
+import com.aws.greengrass.clientdevices.auth.iot.events.VerifyClientDeviceIdentityEvent;
```

Comment:
?? shouldn't this cause a pmd violation? is it being used?

Reply:
I'm using it down in `verifyClientDeviceIdentity()` to emit the event